ChurchCRM/CRM
# Fix Sunday School Parent Display Non-Traditional Roles

#### What's this PR do?
Adjusts the Sunday School Class view so "mother" and "Father" contact info will display correctly(?) in a non-traditional family where a "Head of Household" is _not_ male.

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/11679900/34463846-715cd2e6-ee37-11e7-9006-e6b924d8bb99.png)

#### What Issues does it Close?

Closes #1399 

#### What are the relevant tickets?

#### Any background context you want to provide?

#### Where should the reviewer start?
Note that we're adjusting the SQL query so that "Dad" is a male who's "Family Role" is 1 OR 2 (Head of Household _or_ spouse)

#### How should this be manually tested?
1) Load the Demo Database, and open the "Angels" Class Dashboard: /sundayschool/SundaySchoolClassView.php?groupId=1
2) note that this PR populates the Mom and Dad information for Herminia Hart, and for Jean Hart (where Constance is a Female Head of Household, and where Marion is a Male Spouse")  These values were not previously populated

#### How should the automated tests treat this?

#### Questions:
- Is there a related website / article to substantiate / explain this change?
  -  [x ] No

- Does the development wiki need an update?
  -  [x ] No

- Does the user documentation wiki need an update?
  -  [x ] No

- Does this add new dependencies?
  -  [x ] No

- Does this need to add new data to the demo database
  -  [x ] No
